app/src/main/java/com/github/sdp/ratemyepfl/model/user/User.kt

```diff
@@ -2,13 +2,16 @@ package com.github.sdp.ratemyepfl.model.user
 
 import com.github.sdp.ratemyepfl.auth.ConnectedUser
 import com.github.sdp.ratemyepfl.database.UserRepository
+import com.github.sdp.ratemyepfl.model.items.Class
+import com.google.gson.Gson
 import kotlinx.serialization.Serializable
 
 @Serializable
 data class User(
     val uid: String,
     val username: String?,
     val email: String?,
+    val timetable: ArrayList<Class>? = ArrayList<Class>()
```

Comment:
Maybe we could have a class Timetable instead of this ArrayList.

Reply:
Sure !